proto-public/pbmesh/v1alpha1/pbproxystate/traffic_permissions.proto

```diff
@@ -0,0 +1,46 @@
+// Copyright (c) HashiCorp, Inc.
+// SPDX-License-Identifier: BUSL-1.1
+
+syntax = "proto3";
+
+package hashicorp.consul.mesh.v1alpha1.pbproxystate;
+
+message L7TrafficPermission {}
+
+enum TrafficPermissionAction {
+  INTENTION_ACTION_DENY = 0;
+  INTENTION_ACTION_ALLOW = 1;
+}
+
+message L4TrafficPermission {
+  TrafficPermissionAction default_action = 1;
```

Comment:
is it called `default_action` as opposed to `action`?

Reply:
This will end up holding Consul's default for intentions (default allow or default deny). It feels like `action` would be a bit confusing because it is dependent on the scope l4 principals too.